pkg/operator/reports.go

```diff
@@ -859,6 +934,57 @@ func (op *Reporting) queueDependentReportsForReport(report *metering.Report) err
 	return nil
 }
 
+func (op *Reporting) doReportsDependOnThisReport(report *metering.Report) bool {
+	// Look for all reports in the namespace
+	reports, err := op.reportLister.Reports(report.Namespace).List(labels.Everything())
+	if err != nil {
+		op.logger.Errorf("was unable to list reports for report: %s, while looking for dependent reports", report.Name)
+		return false
+	}
```

Comment:
We already made the call to grab the list of reports at the call site. Can we just pass the list of reports we grab there as a parameter to this method?

Reply:
yep